packages/lambda/mixin.core.js

```diff
@@ -69,6 +85,54 @@ class LambdaMixin extends Mixin {
   handleArguments(argv) {
     this.options = { ...this.options, ...argv };
   }
+
+  diagnose() {
+    const warnings = [];
+    const targetNodeVersion =
+      !this.config.node || this.config.node === 'current'
+        ? process.version
+        : this.config.node;
+
+    if (
+      semver.gt(
+        semver.coerce(targetNodeVersion),
+        semver.coerce(MAX_NODE_VERSION)
+      )
+    ) {
+      warnings.push(
+        'AWS Lambda only supports Node.js up to version:',
+        MAX_NODE_VERSION
+      );
+      warnings.push(
+        'Please specify or use a Node.js version lower than or equal to this',
```

Comment:
These lines should not be separate warning, but rather concatenated - this applies to all similar occurrences in this file...